assets/newspack-ui/scss/variables/_colors.scss

```diff
@@ -0,0 +1,48 @@
+:root {
+	// Newspack Blues
+	--newspack-ui-color-primary-000: #f5fdff;
+	--newspack-ui-color-primary-050: #d6f6ff;
+	--newspack-ui-color-primary-100: #b4e4ff;
+	--newspack-ui-color-primary-200: #93ccfd;
+	--newspack-ui-color-primary-300: #72affb;
+	--newspack-ui-color-primary-400: #528dfc;
+	--newspack-ui-color-primary-500: #36f;
+	--newspack-ui-color-primary-600: #2240d5;
+	--newspack-ui-color-primary-700: #1522af;
+	--newspack-ui-color-primary-800: #0b0b8d;
+	--newspack-ui-color-primary-900: #0d046e;
+	--newspack-ui-color-primary-950: #0e0052;
+
+	// Grays - notes from the Gutenberg styles.
+	// TODO: Import from 'wordpress/base-styles/colors' ?
+	--newspack-ui-color-black: #000;		// Use only when you truly need pure black. For UI, use $gray-900.
+	--newspack-ui-color-gray-900: #1e1e1e;
+	--newspack-ui-color-gray-800: #2f2f2f;
+	--newspack-ui-color-gray-700: #757575;	// Meets 4.6:1 text contrast against white.
+	--newspack-ui-color-gray-600: #949494;	// Meets 3:1 UI or large text contrast against white.
+	--newspack-ui-color-gray-400: #ccc;
+	--newspack-ui-color-gray-300: #ddd;		// Used for most borders.
+	--newspack-ui-color-gray-200: #e0e0e0;	// Used sparingly for light borders.
+	--newspack-ui-color-gray-100: #f0f0f0;	// Used for light gray backgrounds.
+	--newspack-ui-color-white: #fff;
+
+	// Alert colors
+	--newpsack-ui-color-alert-yellow: #f0b849;
+	--newspack-ui-color-alert-green: #4ab866;
+	--newspack-ui-color-alert-red: #cc1818;
+	// --newpsack-ui-color-alert-yellow-bg: TBD
+	--newspack-ui-color-alert-green-bg: #edfaef;
+	--newpsack-ui-color-alert-red-bg: #fcf0f1;
+
+	// Specific assignments - general:
+	--newspack-ui-color-text-high-contrast: var( --newspack-ui-color-gray-900 );
+	--newspack-ui-color-text-med-contrast: var( --newspack-ui-color-gray-700 );
+	--newspack-ui-color-border: var( --newspack-ui-color-gray-300 );
+	--newspack-ui-color-body-bg: var( --newspack-ui-color-white );
+
+	// Specific assignments - form elements
+	--newspack-ui-color-button-bg: var( --newspack-ui-color-gray-900 );
+	--newspack-ui-color-button-bg-hover: var( --newspack-ui-color-gray-800 ); // TODO: Replace w/correct value.
+	--newspack-ui-color-button-text: var( --newspack-ui-color-white );
+	--newspack-ui-color-radio-bg: var( --newspack-ui-color-gray-900 );
```

Comment:
Shouldn't this be `--newspack-ui-color-primary-500`?

Ref:

<img width="223" alt="image" src="https://github.com/Automattic/newspack-plugin/assets/820752/aa00d200-64a3-4b7d-8927-b6855f984052">

Reply:
This stemmed from my confusion around how we're handling the colours! Followed up about that here (p1704831597129219-slack-newspack-reader-activation) for this comment and the previous one -- thanks for pointing these out, I kind of forgot about them! 😅

Reply:
@miguelpeixe Regarding the above, we'll be going forward with the brand colour for primary actions and black/grey as a fallback -- does it sound okay to update everything to use the black/greys for now?

Reply:
The above has been done in [3bb8809](https://github.com/Automattic/newspack-plugin/pull/2810/commits/3bb8809a9c40527e54dcf1cb709812bef3a9f051) kind of as a temporary state 🙂

Reply:
Awesome! Thank you for looking further into this. I like the approach with the temporary state, which might turn out to be the permanent state 🤷